### PR TITLE
Added determinate function

### PR DESCRIPTION
**Pull request by codecalec**
Hi.
This is my first pull request so just let me know if there is anything else I should do. I thought the linear algebra class was missing a pretty important aspect of real valued matrices so thought it should be included.

It uses Laplace expansion to find the determinate recursively. It a pretty slow algorithm but it does the trick.

### CODE REVIEW

File: linear_algebra/src/tests.py [hunk 1]
```diff
@@ -118,6 +118,13 @@ def test_str_matrix(self):
         A = Matrix([[1, 2, 3], [2, 4, 5], [6, 7, 8]], 3, 3)
         self.assertEqual("|1,2,3|\n|2,4,5|\n|6,7,8|\n", str(A))
 
+    def test_determinate(self):
+        """
+            test for determinate()
+        """
+        A = Matrix([[1, 1, 4, 5], [3, 3, 3, 2], [5, 1, 9, 0], [9, 7, 7, 9]], 4, 4)
+        self.assertEqual(-376, A.determinate())
+
```
[the remark this answers is not in the document]
Comment (cclauss) [reply linear_algebra/src/tests.py]: We can see that our tests are not being run:
https://travis-ci.com/TheAlgorithms/Python/builds/133339192#L386

[the remark this answers is not in the document]
Comment (codecalec) [reply linear_algebra/src/tests.py]: Added

**Comment by codecalec** [comment]
Awesome. Sorry for the hiccups

**Comment by cclauss** [comment]
No problems.

**Comment by cclauss** [comment]
Please give some thought to https://github.com/TheAlgorithms/Python/pull/1124#issuecomment-545826817

**Pull request merged**